bin/anthology/sigs.py

```diff
@@ -56,6 +55,11 @@ def load_from_dir(self, directory):
                 sig = SIG.from_dict(data)
                 self.sigs[sig.acronym] = sig
 
+    def remove_volume(self, full_volume_id):
+        for acronym, sig in self.sigs.items():
+            # print(f'Removing {full_volume_id} from {acronym}')
```

Comment:
I would like to either have a debug call here or the commented out print call completely removed.

Reply:
Will do!